client/components/auth/SigninForm.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { PropTypes } from 'prop-types';
+import requireauth from '../../utils/requireAuth';
+import { login } from '../../actions/authActions';
+import validateInput from '../../../server/shared/validations/signin';
+import TextFieldGroup from '../common/TextFieldGroup';
+import Footer from '../common/footer.jsx';
```

Comment:
You have saved your component with lowercase, and import it with uppercase . I really think React component should be saved with PascalCase (Capital letter first) than what you did here. Consider